internal/db/db.go

```diff
@@ -30,7 +31,7 @@ import (
 	"github.com/sourcenetwork/defradb/acp/identity"
 	"github.com/sourcenetwork/defradb/client"
 	"github.com/sourcenetwork/defradb/datastore"
-	"github.com/sourcenetwork/defradb/errors"
+	defraErrors "github.com/sourcenetwork/defradb/errors"
```

Comment:
todo: Please add the `Join` function to the defra `errors` package like we did for `Is` and only import the defra `errors` package.